web/frontend/tsconfig.json

```diff
@@ -13,5 +13,15 @@
 
     "noPropertyAccessFromIndexSignature": false,
     "exactOptionalPropertyTypes": false,
-  }
+  },
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.js",
+    "src/**/*.d.ts",
```

Comment:
This is redundant with `"src/**/*.ts"` above

Reply:
ah duh 🤦 fixed!